Sources/General/Commands/Setup.swift

```diff
@@ -8,15 +8,31 @@ import ZIPFoundation
 
 final class Setup: ParsableCommand {
 
-    enum Error: Swift.Error {
+    enum Error: Swift.Error, CustomStringConvertible {
```

Comment:
Isn't it better to use `LocalizedError` instead of `CustomStringConvertible` for errors?

Reply:
`ArgumentParser` will not print `LocalizedError` properly